test/countries/test_bosnia_and_herzegovina.py

```diff
@@ -10,285 +10,281 @@
 #  License: MIT (see LICENSE file)
 #  Copyright: Kateryna Golovanova <kate@kgthreads.com>, 2022
 
-import unittest
-from datetime import date
+from holidays.countries.bosnia_and_herzegovina import BosniaAndHerzegovina, BA
+from holidays.countries.bosnia_and_herzegovina import BIH
+from test.common import TestCase
 
-import holidays
 
-
-class TestBA(unittest.TestCase):
+class TestBA(TestCase):
     def setUp(self):
-        self.holidays = holidays.BA(observed=False)
-        self.holidays_observed = holidays.BA(observed=True)
-
-    def test_new_years(self):
-        for dt in (date(2010, 12, 31), date(2017, 1, 3)):
-            self.assertNotIn(dt, self.holidays)
+        self.holidays = BosniaAndHerzegovina(observed=False)
```

Comment:
Not sure why we're testing non-default (observed=False) behaviour here.

Reply:
Before it was like that (`self.holidays = holidays.BA(observed=False) …`), so I left it. :)